Makefile

```diff
@@ -272,6 +272,20 @@ dev: prebuild-check deps generate $(FRESH_BIN)
 	docker-compose up -d db auth
 	F8_DEVELOPER_MODE_ENABLED=true $(FRESH_BIN)
 
+.PHONY: core-dev-os
```

Comment:
let's call it `dev-openshift`

Reply:
sure! thanks @sbose78

Reply:
Shouldn't we use make dev for this and get rid of the compose thing completely?

Reply:
@alexeykazakov yeah that's the plan.
1. Start with having a separate make target for folks to try out and stabilize this thing ( since we would be one of the first to use the OpenShift features of kedge )
2. After we are happy - and all's stable, we will make this `make dev` and remove Docker Compose altogether.

Reply:
@sbose78 done in here https://github.com/fabric8-services/fabric8-wit/pull/1741/commits/ea3bbdcb14d6409ccf9b53d78364bd90e621ae21
I have renamed it to `dev-wit-openshift` and next target will be `dev-planner-openshift` that will run all 4 dependencies in minishift
WDYT?